softlayer/vm/softlayer_virtual_guest_creator.go

```diff
@@ -118,6 +118,8 @@ func (c *softLayerVirtualGuestCreator) createBySoftlayer(agentID string, stemcel
 		}
 	}
 
+	vm.ConfigureNetworks2(networks)
```

Comment:
Where's the error check? And why is this named such an awkward name? The name should reflect the style of network configuration that's being done.